packages/radio-group/theme/lumo/vaadin-radio-button-styles.js

```diff
@@ -40,6 +40,10 @@ registerStyles(
       cursor: var(--lumo-clickable-cursor);
     }
 
+    ::slotted(input) {
+      width: calc(var(--lumo-size-m) / 2 + var(--lumo-space-s));
+    }
```

Comment:
Is the purpose of this to cover the space between the input and the label? I don't think that’s absolutely needed, but perhaps it’s nice to have.

Reply:
Removed.